src/Native/Runtime/module.cpp

```diff
@@ -1270,3 +1270,49 @@ UInt32 GenericInstanceDesc::DacSize(TADDR addr)
     return GenericInstanceDesc::GetSize(dummyDesc.GetFlags(), arity);
 }
 #endif // DACCESS_COMPILE
+
+/* static */
+ModuleInfo * ModuleInfo::Create(void * pHeaderStart, void * pHeaderEnd)
```

Comment:
Sounds good to me. (We will likely invent a different name as we reconcile it with readytorun format - it is called READYTORUN_SECTION in CoreCLR readytorun.h right now.)